Executable dependent on chrome

Hi,
I wanted to try the new released build 0.0.1 for Windows.
Sadly, the application crashes on startup. It seems to me that it is related to Google Chrome not being installed on my Computer.
Here is the error message:

```
2023-05-04 20:03:47 [ERROR] Uncaught exception:
    Traceback (most recent call last):
      File "usdb_syncer\main.py", line 36, in <module>
      File "usdb_syncer\main.py", line 31, in cli_entry
      File "usdb_syncer\gui\mw.py", line 306, in main
      File "usdb_syncer\gui\mw.py", line 315, in _load_main_window
      File "usdb_syncer\song_list_fetcher.py", line 23, in get_all_song_data
      File "usdb_syncer\song_list_fetcher.py", line 49, in get_available_songs
      File "usdb_syncer\usdb_scraper.py", line 333, in get_usdb_available_songs
      File "usdb_syncer\usdb_scraper.py", line 231, in get_usdb_page
      File "usdb_syncer\usdb_scraper.py", line 222, in page
      File "usdb_syncer\usdb_scraper.py", line 78, in session
      File "usdb_syncer\usdb_scraper.py", line 64, in new_session_with_cookies
      File "usdb_syncer\settings.py", line 220, in cookies
      File "browser_cookie3\__init__.py", line 973, in chrome
      File "browser_cookie3\__init__.py", line 516, in __init__
      File "browser_cookie3\__init__.py", line 331, in __init__
      File "browser_cookie3\__init__.py", line 378, in __add_key_and_cookie_file
    browser_cookie3.BrowserCookieError: Failed to find Chrome cookie
[15584] Failed to execute script 'main' due to unhandled exception!
```

It would be nice, if the program could start, even if no chrome cookies could be found.

--- a/.github/workflows/release.yaml
+++ b/.github/workflows/release.yaml
@@ -36,6 +36,8 @@ jobs:
         run: |
           python -m pip install --upgrade pip wheel
           python -m pip install '.[dev]'
+      - name: Write version and commit hash
+        run: write_release_info -p src/usdb_syncer/constants.py -v ${{ github.ref_name }} -c ${{ github.workflow_sha }}
       - name: Generate GUI elements
         run: generate_pyside_files
       - name: Scrape USDB song list to ship with the bundle

--- a/.github/workflows/release.yaml
+++ b/.github/workflows/release.yaml
@@ -31,7 +31,7 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v4
         with:
-          python-version: "3.10"
+          python-version: "3.11"
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip wheel

--- a/.github/workflows/tox.yaml
+++ b/.github/workflows/tox.yaml
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.10"]
+        python-version: ["3.11"]
 
     steps:
       - uses: actions/checkout@v3

--- a/.vscode-dist/settings.json
+++ b/.vscode-dist/settings.json
@@ -6,7 +6,8 @@
   "python.formatting.provider": "black",
   "editor.formatOnSave": true,
   "editor.codeActionsOnSave": {
-    "source.organizeImports": true
+    "source.organizeImports": true,
+    "source.fixAll": true
   },
   "editor.rulers": [
     88

--- a/README.md
+++ b/README.md
@@ -9,8 +9,8 @@
 ## Development
 
 **USDB Syncer** is written in Python.
-The following explains how to set up a development environment. A Python installation with
-version 3.10 or later is assumed.
+The following explains how to set up a development environment.
+A Python 3.11 installation is assumed.
 
 ### Python Setup
 
@@ -49,11 +49,11 @@ The following tools are part of the test pipeline:
 
 If you don’t want to run the whole test pipeline, you can also use single commands from the pipeline, e.g., `pytest`. The tools will automatically pick up the correct configuration from the `pyproject.toml` file.
 
-
 ### Building
 
 **USDB Syncer** uses [pyinstaller](https://github.com/pyinstaller/pyinstaller) for creating
 executables. In order to build **USDB Syncer**, run the following command
+
 ```bash
 # On Linux
 pyinstaller --name "USDBSyncer" --onefile src/usdb_syncer/main.py
@@ -65,7 +65,6 @@ pyinstaller --name "USDBSyncer" --windowed src/usdb_syncer/main.py
 pyinstaller --name "USDBSyncer" --onefile src/usdb_syncer/main.py
 ```
 
-
 ## Versioning
 
 **USDB Syncer** uses [semantic versioning (semver)](https://semver.org/) as versioning scheme.
@@ -73,10 +72,9 @@ However, since **USDB Syncer** is not a library/API but a user-facing applicatio
 versions according to the following scheme:
 
 - `MAJOR` version increments mean a breaking change for the end user, be it the need to install additional
-(3rd party) tools or changes that make it necessary to make changes to the already downloaded songs.
+  (3rd party) tools or changes that make it necessary to make changes to the already downloaded songs.
 - `MINOR` version increments only involve adding backward compatible features.
 - `PATCH` version increments bring bugfixes.
-
 
 We will try to avoid `MAJOR` version increments whenever possible, but since the project is still in the
 startup phase, they cannot be completely ruled out.

--- a/setup.cfg
+++ b/setup.cfg
@@ -46,10 +46,10 @@ dev =
     %(build)s
 
 lint =
-    isort
-    black
-    mypy
-    pylint==2.15
+    isort==5.12
+    black==23.3
+    mypy==1.2
+    pylint==2.17
     pytest
     types-setuptools
     types-requests

--- a/setup.cfg
+++ b/setup.cfg
@@ -9,7 +9,7 @@ url = https://github.com/bohning/usdb_syncer
 
 
 [options]
-python_requires = >=3.10
+python_requires = ==3.11.*
 packages = find:
 package_dir =
     =src

--- a/setup.cfg
+++ b/setup.cfg
@@ -66,3 +66,4 @@ console_scripts =
     usdb_syncer = usdb_syncer.main:cli_entry
     generate_pyside_files = tools.generate_pyside_files:cli_entry
     generate_song_list_json = tools.generate_song_list_json:cli_entry
+    write_release_info = tools.write_release_info:cli_entry

--- a/src/tools/__init__.py
+++ b/src/tools/__init__.py
@@ -2,3 +2,4 @@
 
 from tools.generate_pyside_files import main as generate_pyside_files
 from tools.generate_song_list_json import main as generate_song_list_json
+from tools.write_release_info import main as write_release_info

--- a/src/tools/write_release_info.py
+++ b/src/tools/write_release_info.py
@@ -1,0 +1,29 @@
+"""Replaces version and commit placeholders in constants.py."""
+
+import argparse
+from pathlib import Path
+
+
+def main(path: Path, version: str, commit_hash: str) -> None:
+    print(f"Rewriting file {path}")
+    old = path.read_text(encoding="utf8")
+    new = old.replace('VERSION = "dev"', f'VERSION = "{version}"').replace(
+        'COMMIT_HASH = "dev"', f'COMMIT_HASH = "{commit_hash}"'
+    )
+    path.write_text(new, encoding="utf8", newline="\n")
+    print(f"New content:\n{path.read_text(encoding='utf8')}")
+
+
+def cli_entry() -> None:
+    parser = argparse.ArgumentParser(
+        description="Replaces version and commit placeholders in constants.py."
+    )
+    parser.add_argument("--path", "-p", help="the file with the variables to replace")
+    parser.add_argument("--version", "-v")
+    parser.add_argument("--commit", "-c", help="commit hash")
+    args = parser.parse_args()
+    main(Path(args.path), args.version, args.commit)
+
+
+if __name__ == "__main__":
+    cli_entry()

--- a/src/usdb_syncer/constants.py
+++ b/src/usdb_syncer/constants.py
@@ -2,6 +2,10 @@
 
 import re
 
+# set in the release workflow
+VERSION = "dev"
+COMMIT_HASH = "dev"
+
 MINIMUM_BPM = 200.0
 
 

--- a/src/usdb_syncer/gui/about_dialog.py
+++ b/src/usdb_syncer/gui/about_dialog.py
@@ -2,6 +2,7 @@
 
 from PySide6.QtCore import QTimer
 from PySide6.QtWidgets import QDialog, QWidget
+from usdb_syncer.constants import COMMIT_HASH, VERSION
 
 from usdb_syncer.gui.forms.AboutDialog import Ui_Dialog
 
@@ -16,6 +17,8 @@ class AboutDialog(Ui_Dialog, QDialog):
     def __init__(self, parent: QWidget) -> None:
         super().__init__(parent=parent)
         self.setupUi(self)
+        self.label_version.setText(VERSION)
+        self.label_commit.setText(COMMIT_HASH)
         self._reset_text()
 
     def _scroll_down(self) -> None:

--- a/src/usdb_syncer/gui/about_dialog.py
+++ b/src/usdb_syncer/gui/about_dialog.py
@@ -1,0 +1,13 @@
+"""Dialog with app settings."""
+
+from PySide6.QtWidgets import QDialog, QWidget
+
+from usdb_syncer.gui.forms.AboutDialog import Ui_Dialog
+
+
+class AboutDialog(Ui_Dialog, QDialog):
+    """Dialog with app settings."""
+
+    def __init__(self, parent: QWidget) -> None:
+        super().__init__(parent=parent)
+        self.setupUi(self)

--- a/src/usdb_syncer/gui/about_dialog.py
+++ b/src/usdb_syncer/gui/about_dialog.py
@@ -2,8 +2,8 @@
 
 from PySide6.QtCore import QTimer
 from PySide6.QtWidgets import QDialog, QWidget
-from usdb_syncer.constants import COMMIT_HASH, VERSION
 
+from usdb_syncer.constants import COMMIT_HASH, VERSION
 from usdb_syncer.gui.forms.AboutDialog import Ui_Dialog
 
 RESET_GAP = 4000

--- a/src/usdb_syncer/gui/about_dialog.py
+++ b/src/usdb_syncer/gui/about_dialog.py
@@ -55,8 +55,8 @@ class AboutDialog(Ui_Dialog, QDialog):
         credits_text += "<b>Copyright © 2023</b></center>"
 
         self.credits.setHtml(credits_text)
-        self.credits.setHorizontalScrollBarPolicy(Qt.ScrollBarAlwaysOff)
-        self.credits.setVerticalScrollBarPolicy(Qt.ScrollBarAlwaysOff)
+        self.credits.setHorizontalScrollBarPolicy(Qt.ScrollBarPolicy.ScrollBarAlwaysOff)
+        self.credits.setVerticalScrollBarPolicy(Qt.ScrollBarPolicy.ScrollBarAlwaysOff)
 
         self._reset_text()
 

--- a/src/usdb_syncer/gui/about_dialog.py
+++ b/src/usdb_syncer/gui/about_dialog.py
@@ -1,6 +1,6 @@
 """Dialog with app settings."""
 
-from PySide6.QtCore import Qt, QTimer
+from PySide6.QtCore import QTimer
 from PySide6.QtWidgets import QDialog, QWidget
 
 from usdb_syncer.gui.forms.AboutDialog import Ui_Dialog
@@ -16,48 +16,6 @@ class AboutDialog(Ui_Dialog, QDialog):
     def __init__(self, parent: QWidget) -> None:
         super().__init__(parent=parent)
         self.setupUi(self)
-        credits_text = ""
-        credits_text += "<style>a { color: #4479D4 text-decoration: none}</style>"
-        credits_text += "<center><br><br><br>"
-        credits_text += "<b>Thank you</b> for using USDB Syncer!<br><br>"
-        credits_text += (
-            "Keep your karaoke song collection up to date at the click of a button."
-        )
-        credits_text += "<br><br>***<br><br>"
-        credits_text += "<b>More info</b>"
-        credits_text += "<br><a href='https://github.com/bohning/usdb_syncer/wiki'>USDB Syncer Wiki</a><br>"
-        credits_text += "<br><br>***<br><br>"
-        credits_text += "<b>Support</b>"
-        credits_text += "<br>You can support the developers by buying them some <a href='https://www.buymeacoffee.com/usdbsyncer'>vegan pizza!</a><br>"
-        credits_text += "<br><br>***<br><br>"
-        credits_text += "<b>Main Programmers</b>"
-        credits_text += "<br>bohning<br>Ultorex<br><br>"
-        credits_text += "<b>Contributing Programmers</b>"
-        credits_text += "<br>mjhalwa<br>g3n35i5<br>BWagener"
-        credits_text += "<br><br>***<br><br>"
-        credits_text += "<b>Application Icon</b>"
-        credits_text += "<br>rawpixel.com on Freepik<br>"
-        credits_text += "Website:"
-        credits_text += ' <a href="https://www.freepik.com/free-vector/pink-neon-cloud-icon-digital-networking-system_16406257.htm">freepik.com</a><br><br>'
-        credits_text += "<b>Fugue Icons</b>"
-        credits_text += "<br>© 2021 Yusuke Kamiyamane<br>"
-        credits_text += "Website:"
-        credits_text += (
-            ' <a href="https://p.yusukekamiyamane.com/">p.yusukekamiyamane.com/</a><br>'
-        )
-        credits_text += "License:"
-        credits_text += ' <a href="http://creativecommons.org/licenses/by/3.0">CC Attribution 3.0 Unported</a>'
-        credits_text += "<br><br>***<br><br>"
-        credits_text += "<b>Tester</b>"
-        credits_text += "<br>Rakuri"
-        credits_text += "<br>Hoanzl"
-        credits_text += "<br>GrüneNeun<br><br><br><br><br>"
-        credits_text += "<b>Copyright © 2023</b></center>"
-
-        self.credits.setHtml(credits_text)
-        self.credits.setHorizontalScrollBarPolicy(Qt.ScrollBarPolicy.ScrollBarAlwaysOff)
-        self.credits.setVerticalScrollBarPolicy(Qt.ScrollBarPolicy.ScrollBarAlwaysOff)
-
         self._reset_text()
 
     def _scroll_down(self) -> None:

--- a/src/usdb_syncer/gui/about_dialog.py
+++ b/src/usdb_syncer/gui/about_dialog.py
@@ -1,13 +1,74 @@
 """Dialog with app settings."""
 
+from PySide6.QtCore import Qt, QTimer
 from PySide6.QtWidgets import QDialog, QWidget
 
 from usdb_syncer.gui.forms.AboutDialog import Ui_Dialog
 
+RESET_GAP = 4000
+SCROLL_STEP = 1
+SCROLL_GAP = 100
+
 
 class AboutDialog(Ui_Dialog, QDialog):
-    """Dialog with app settings."""
+    """Dialog with about info and credits."""
 
     def __init__(self, parent: QWidget) -> None:
         super().__init__(parent=parent)
         self.setupUi(self)
+        credits_text = ""
+        credits_text += "<style>a { color: #4479D4 text-decoration: none}</style>"
+        credits_text += "<center><br><br><br>"
+        credits_text += "<b>Thank you</b> for using USDB Syncer!<br><br>"
+        credits_text += (
+            "Keep your karaoke song collection up to date at the click of a button."
+        )
+        credits_text += "<br><br>***<br><br>"
+        credits_text += "<b>More info</b>"
+        credits_text += "<br><a href='https://github.com/bohning/usdb_syncer/wiki'>USDB Syncer Wiki</a><br>"
+        credits_text += "<br><br>***<br><br>"
+        credits_text += "<b>Support</b>"
+        credits_text += "<br>You can support the developers by buying them some <a href='https://www.buymeacoffee.com/usdbsyncer'>vegan pizza!</a><br>"
+        credits_text += "<br><br>***<br><br>"
+        credits_text += "<b>Main Programmers</b>"
+        credits_text += "<br>bohning<br>Ultorex<br><br>"
+        credits_text += "<b>Contributing Programmers</b>"
+        credits_text += "<br>mjhalwa<br>g3n35i5<br>BWagener"
+        credits_text += "<br><br>***<br><br>"
+        credits_text += "<b>Application Icon</b>"
+        credits_text += "<br>rawpixel.com on Freepik<br>"
+        credits_text += "Website:"
+        credits_text += ' <a href="https://www.freepik.com/free-vector/pink-neon-cloud-icon-digital-networking-system_16406257.htm">freepik.com</a><br><br>'
+        credits_text += "<b>Fugue Icons</b>"
+        credits_text += "<br>© 2021 Yusuke Kamiyamane<br>"
+        credits_text += "Website:"
+        credits_text += (
+            ' <a href="https://p.yusukekamiyamane.com/">p.yusukekamiyamane.com/</a><br>'
+        )
+        credits_text += "License:"
+        credits_text += ' <a href="http://creativecommons.org/licenses/by/3.0">CC Attribution 3.0 Unported</a>'
+        credits_text += "<br><br>***<br><br>"
+        credits_text += "<b>Tester</b>"
+        credits_text += "<br>Rakuri"
+        credits_text += "<br>Hoanzl"
+        credits_text += "<br>GrüneNeun<br><br><br><br><br>"
+        credits_text += "<b>Copyright © 2023</b></center>"
+
+        self.credits.setHtml(credits_text)
+        self.credits.setHorizontalScrollBarPolicy(Qt.ScrollBarAlwaysOff)
+        self.credits.setVerticalScrollBarPolicy(Qt.ScrollBarAlwaysOff)
+
+        self._reset_text()
+
+    def _scroll_down(self) -> None:
+        scroll_bar = self.credits.verticalScrollBar()
+        if scroll_bar.value() >= scroll_bar.maximum():
+            QTimer.singleShot(RESET_GAP, self._reset_text)
+        else:
+            scroll_bar.setValue(scroll_bar.value() + SCROLL_STEP)
+            QTimer.singleShot(SCROLL_GAP, self._scroll_down)
+
+    def _reset_text(self) -> None:
+        scroll_bar = self.credits.verticalScrollBar()
+        scroll_bar.setValue(scroll_bar.minimum())
+        QTimer.singleShot(SCROLL_GAP, self._scroll_down)

--- a/src/usdb_syncer/gui/forms/AboutDialog.ui
+++ b/src/usdb_syncer/gui/forms/AboutDialog.ui
@@ -1,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>Dialog</class>
+ <widget class="QDialog" name="Dialog">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>500</width>
+    <height>300</height>
+   </rect>
+  </property>
+  <property name="windowTitle">
+   <string>USDB Syncer</string>
+  </property>
+  <property name="styleSheet">
+   <string notr="true">background-image: url(:/splash/splash.png);</string>
+  </property>
+ </widget>
+ <resources>
+  <include location="../resources/resources.qrc"/>
+ </resources>
+ <connections/>
+</ui>

--- a/src/usdb_syncer/gui/forms/AboutDialog.ui
+++ b/src/usdb_syncer/gui/forms/AboutDialog.ui
@@ -76,14 +76,23 @@ date at the click of a button.
 
 ***
 
-**More info**
-
-[USDB Syncer Wiki](https://github.com/bohning/usdb_syncer/wiki)
+Licensed under the 
+[MIT License](https://mit-license.org/).
+Find the source code on 
+[GitHub](https://github.com/bohning/usdb_syncer/).
 
 
 ***
 
-**Support
+
+**More info**
+[USDB Syncer Wiki](https://github.com/bohning/usdb_syncer/wiki)
+
+
+
+***
+
+**Support**
 You can support the developers by buying them some 
 [vegan pizza!](https://www.buymeacoffee.com/usdbsyncer)
 
@@ -135,7 +144,9 @@ hr { height: 1px; border-width: 0; }
 li.unchecked::marker { content: &quot;\2610&quot;; }
 li.checked::marker { content: &quot;\2612&quot;; }
 &lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
-&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;br /&gt;&lt;br /&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Thank you&lt;/span&gt; for using USDB Syncer!&lt;br /&gt;&lt;br /&gt;Keep your karaoke song collection up to date at the click of a button.&lt;br /&gt;&lt;br /&gt;***&lt;br /&gt;&lt;br /&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;More info&lt;/span&gt;&lt;br /&gt;&lt;a href=&quot;https://github.com/bohning/usdb_syncer/wiki&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#00aeef;&quot;&gt;USDB Syncer Wiki&lt;/span&gt;&lt;/a&gt;&lt;br /&gt;&lt;br /&gt;&lt;br /&gt;***&lt;br /&gt;&lt;br /&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Support&lt;/span&gt;&lt;br /&gt;You can support the developers by buying them some &lt;a href=&quot;https://www.buymeacoffee.com/usdbsyncer&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#00aeef;&quot;&gt;vegan pizza!&lt;/span&gt;&lt;/a&gt;&lt;br /&gt;&lt;br /&gt;&lt;br /&gt;***&lt;br /&gt;&lt;br /&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Main Programmers&lt;/span&gt;&lt;br /&gt;bohning&lt;br /&gt;RumovZ&lt;br /&gt;&lt;br /&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Contributing Programmers&lt;/span&gt;&lt;br /&gt;mjhalwa&lt;br /&gt;g3n35i5&lt;br /&gt;BWagener&lt;br /&gt;&lt;br /&gt;***&lt;br /&gt;&lt;br /&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Application Icon&lt;/span&gt;&lt;br /&gt;rawpixel.com on Freepik&lt;br /&gt;Website: &lt;a href=&quot;https://www.freepik.com/free-vector/pink-neon-cloud-icon-digital-networking-system_16406257.htm&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#00aeef;&quot;&gt;freepik.com&lt;/span&gt;&lt;/a&gt;&lt;br /&gt;&lt;br /&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Fugue Icons&lt;/span&gt;&lt;br /&gt;© 2021 Yusuke Kamiyamane&lt;br /&gt;Website: &lt;a href=&quot;https://p.yusukekamiyamane.com/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#00aeef;&quot;&gt;p.yusukekamiyamane.com/&lt;/span&gt;&lt;/a&gt;&lt;br /&gt;License: &lt;a href=&quot;http://creativecommons.org/licenses/by/3.0&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#00aeef;&quot;&gt;CC Attribution 3.0 Unported&lt;/span&gt;&lt;/a&gt;&lt;br /&gt;&lt;br /&gt;***&lt;br /&gt;&lt;br /&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Testers&lt;/span&gt;&lt;br /&gt;Rakuri&lt;br /&gt;Hoanzl&lt;br /&gt;GrüneNeun&lt;br /&gt;&lt;br /&gt;&lt;br /&gt;&lt;br /&gt;&lt;br /&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Copyright © 2023&lt;/span&gt;&lt;/p&gt;
+&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;br /&gt;&lt;br /&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Thank you&lt;/span&gt; for using USDB Syncer!&lt;br /&gt;&lt;br /&gt;Keep your karaoke song collection up to date at the click of a button.&lt;br /&gt;&lt;br /&gt;***&lt;br /&gt;&lt;br /&gt;Licensed under the &lt;a href=&quot;https://mit-license.org/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#00aeef;&quot;&gt;MIT License&lt;/span&gt;&lt;/a&gt;.&lt;br /&gt;Find the source code on &lt;a href=&quot;https://github.com/bohning/usdb_syncer/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#00aeef;&quot;&gt;GitHub&lt;/span&gt;&lt;/a&gt;.&lt;br /&gt;&lt;br /&gt;&lt;br /&gt;***&lt;/p&gt;
+&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;More info&lt;/span&gt;&lt;br /&gt;&lt;a href=&quot;https://github.com/bohning/usdb_syncer/wiki&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#00aeef;&quot;&gt;USDB Syncer Wiki&lt;/span&gt;&lt;/a&gt;&lt;br /&gt;&lt;/p&gt;
+&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;***&lt;br /&gt;&lt;br /&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Support&lt;/span&gt;&lt;br /&gt;You can support the developers by buying them some &lt;a href=&quot;https://www.buymeacoffee.com/usdbsyncer&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#00aeef;&quot;&gt;vegan pizza!&lt;/span&gt;&lt;/a&gt;&lt;br /&gt;&lt;br /&gt;&lt;br /&gt;***&lt;br /&gt;&lt;br /&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Main Programmers&lt;/span&gt;&lt;br /&gt;bohning&lt;br /&gt;RumovZ&lt;br /&gt;&lt;br /&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Contributing Programmers&lt;/span&gt;&lt;br /&gt;mjhalwa&lt;br /&gt;g3n35i5&lt;br /&gt;BWagener&lt;br /&gt;&lt;br /&gt;***&lt;br /&gt;&lt;br /&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Application Icon&lt;/span&gt;&lt;br /&gt;rawpixel.com on Freepik&lt;br /&gt;Website: &lt;a href=&quot;https://www.freepik.com/free-vector/pink-neon-cloud-icon-digital-networking-system_16406257.htm&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#00aeef;&quot;&gt;freepik.com&lt;/span&gt;&lt;/a&gt;&lt;br /&gt;&lt;br /&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Fugue Icons&lt;/span&gt;&lt;br /&gt;© 2021 Yusuke Kamiyamane&lt;br /&gt;Website: &lt;a href=&quot;https://p.yusukekamiyamane.com/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#00aeef;&quot;&gt;p.yusukekamiyamane.com/&lt;/span&gt;&lt;/a&gt;&lt;br /&gt;License: &lt;a href=&quot;http://creativecommons.org/licenses/by/3.0&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#00aeef;&quot;&gt;CC Attribution 3.0 Unported&lt;/span&gt;&lt;/a&gt;&lt;br /&gt;&lt;br /&gt;***&lt;br /&gt;&lt;br /&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Testers&lt;/span&gt;&lt;br /&gt;Rakuri&lt;br /&gt;Hoanzl&lt;br /&gt;GrüneNeun&lt;br /&gt;&lt;br /&gt;&lt;br /&gt;&lt;br /&gt;&lt;br /&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Copyright © 2023&lt;/span&gt;&lt;/p&gt;
 &lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'.AppleSystemUIFont'; font-size:13pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="openExternalLinks">

--- a/src/usdb_syncer/gui/forms/AboutDialog.ui
+++ b/src/usdb_syncer/gui/forms/AboutDialog.ui
@@ -59,10 +59,10 @@ padding: 10px;</string>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
-   <property name="horizontalScrollBarPolicy">
+   <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
-   <property name="verticalScrollBarPolicy">
+   <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="markdown">
@@ -92,7 +92,7 @@ You can support the developers by buying them some
 
 **Main Programmers
 bohning
-Ultorex
+RumovZ
 
 **Contributing Programmers**
 mjhalwa
@@ -115,7 +115,7 @@ License:
 ***
 
 
-**Tester**
+**Testers**
 Rakuri
 Hoanzl
 GrüneNeun
@@ -135,7 +135,7 @@ hr { height: 1px; border-width: 0; }
 li.unchecked::marker { content: &quot;\2610&quot;; }
 li.checked::marker { content: &quot;\2612&quot;; }
 &lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
-&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;br /&gt;&lt;br /&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Thank you&lt;/span&gt; for using USDB Syncer!&lt;br /&gt;&lt;br /&gt;Keep your karaoke song collection up to date at the click of a button.&lt;br /&gt;&lt;br /&gt;***&lt;br /&gt;&lt;br /&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;More info&lt;/span&gt;&lt;br /&gt;&lt;a href=&quot;https://github.com/bohning/usdb_syncer/wiki&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;USDB Syncer Wiki&lt;/span&gt;&lt;/a&gt;&lt;br /&gt;&lt;br /&gt;&lt;br /&gt;***&lt;br /&gt;&lt;br /&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Support&lt;/span&gt;&lt;br /&gt;You can support the developers by buying them some &lt;a href=&quot;https://www.buymeacoffee.com/usdbsyncer&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;vegan pizza!&lt;/span&gt;&lt;/a&gt;&lt;br /&gt;&lt;br /&gt;&lt;br /&gt;***&lt;br /&gt;&lt;br /&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Main Programmers&lt;/span&gt;&lt;br /&gt;bohning&lt;br /&gt;Ultorex&lt;br /&gt;&lt;br /&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Contributing Programmers&lt;/span&gt;&lt;br /&gt;mjhalwa&lt;br /&gt;g3n35i5&lt;br /&gt;BWagener&lt;br /&gt;&lt;br /&gt;***&lt;br /&gt;&lt;br /&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Application Icon&lt;/span&gt;&lt;br /&gt;rawpixel.com on Freepik&lt;br /&gt;Website: &lt;a href=&quot;https://www.freepik.com/free-vector/pink-neon-cloud-icon-digital-networking-system_16406257.htm&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;freepik.com&lt;/span&gt;&lt;/a&gt;&lt;br /&gt;&lt;br /&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Fugue Icons&lt;/span&gt;&lt;br /&gt;© 2021 Yusuke Kamiyamane&lt;br /&gt;Website: &lt;a href=&quot;https://p.yusukekamiyamane.com/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;p.yusukekamiyamane.com/&lt;/span&gt;&lt;/a&gt;&lt;br /&gt;License: &lt;a href=&quot;http://creativecommons.org/licenses/by/3.0&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;CC Attribution 3.0 Unported&lt;/span&gt;&lt;/a&gt;&lt;br /&gt;&lt;br /&gt;***&lt;br /&gt;&lt;br /&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Tester&lt;/span&gt;&lt;br /&gt;Rakuri&lt;br /&gt;Hoanzl&lt;br /&gt;GrüneNeun&lt;br /&gt;&lt;br /&gt;&lt;br /&gt;&lt;br /&gt;&lt;br /&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Copyright © 2023&lt;/span&gt;&lt;/p&gt;
+&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;br /&gt;&lt;br /&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Thank you&lt;/span&gt; for using USDB Syncer!&lt;br /&gt;&lt;br /&gt;Keep your karaoke song collection up to date at the click of a button.&lt;br /&gt;&lt;br /&gt;***&lt;br /&gt;&lt;br /&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;More info&lt;/span&gt;&lt;br /&gt;&lt;a href=&quot;https://github.com/bohning/usdb_syncer/wiki&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#00aeef;&quot;&gt;USDB Syncer Wiki&lt;/span&gt;&lt;/a&gt;&lt;br /&gt;&lt;br /&gt;&lt;br /&gt;***&lt;br /&gt;&lt;br /&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Support&lt;/span&gt;&lt;br /&gt;You can support the developers by buying them some &lt;a href=&quot;https://www.buymeacoffee.com/usdbsyncer&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#00aeef;&quot;&gt;vegan pizza!&lt;/span&gt;&lt;/a&gt;&lt;br /&gt;&lt;br /&gt;&lt;br /&gt;***&lt;br /&gt;&lt;br /&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Main Programmers&lt;/span&gt;&lt;br /&gt;bohning&lt;br /&gt;RumovZ&lt;br /&gt;&lt;br /&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Contributing Programmers&lt;/span&gt;&lt;br /&gt;mjhalwa&lt;br /&gt;g3n35i5&lt;br /&gt;BWagener&lt;br /&gt;&lt;br /&gt;***&lt;br /&gt;&lt;br /&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Application Icon&lt;/span&gt;&lt;br /&gt;rawpixel.com on Freepik&lt;br /&gt;Website: &lt;a href=&quot;https://www.freepik.com/free-vector/pink-neon-cloud-icon-digital-networking-system_16406257.htm&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#00aeef;&quot;&gt;freepik.com&lt;/span&gt;&lt;/a&gt;&lt;br /&gt;&lt;br /&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Fugue Icons&lt;/span&gt;&lt;br /&gt;© 2021 Yusuke Kamiyamane&lt;br /&gt;Website: &lt;a href=&quot;https://p.yusukekamiyamane.com/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#00aeef;&quot;&gt;p.yusukekamiyamane.com/&lt;/span&gt;&lt;/a&gt;&lt;br /&gt;License: &lt;a href=&quot;http://creativecommons.org/licenses/by/3.0&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#00aeef;&quot;&gt;CC Attribution 3.0 Unported&lt;/span&gt;&lt;/a&gt;&lt;br /&gt;&lt;br /&gt;***&lt;br /&gt;&lt;br /&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Testers&lt;/span&gt;&lt;br /&gt;Rakuri&lt;br /&gt;Hoanzl&lt;br /&gt;GrüneNeun&lt;br /&gt;&lt;br /&gt;&lt;br /&gt;&lt;br /&gt;&lt;br /&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Copyright © 2023&lt;/span&gt;&lt;/p&gt;
 &lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'.AppleSystemUIFont'; font-size:13pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="openExternalLinks">

--- a/src/usdb_syncer/gui/forms/AboutDialog.ui
+++ b/src/usdb_syncer/gui/forms/AboutDialog.ui
@@ -10,12 +10,78 @@
     <height>300</height>
    </rect>
   </property>
+  <property name="sizePolicy">
+   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
+    <horstretch>0</horstretch>
+    <verstretch>0</verstretch>
+   </sizepolicy>
+  </property>
+  <property name="minimumSize">
+   <size>
+    <width>500</width>
+    <height>300</height>
+   </size>
+  </property>
+  <property name="maximumSize">
+   <size>
+    <width>500</width>
+    <height>300</height>
+   </size>
+  </property>
   <property name="windowTitle">
    <string>USDB Syncer</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/splash/splash.png);</string>
   </property>
+  <widget class="QTextBrowser" name="credits">
+   <property name="geometry">
+    <rect>
+     <x>30</x>
+     <y>130</y>
+     <width>281</width>
+     <height>91</height>
+    </rect>
+   </property>
+   <property name="sizePolicy">
+    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
+     <horstretch>0</horstretch>
+     <verstretch>0</verstretch>
+    </sizepolicy>
+   </property>
+   <property name="styleSheet">
+    <string notr="true">background-image: url();
+background-color: rgba(255, 255, 255, 20);
+color: rgb(255, 255, 255);
+border-radius: 8px;
+padding: 10px;</string>
+   </property>
+   <property name="frameShape">
+    <enum>QFrame::NoFrame</enum>
+   </property>
+   <property name="verticalScrollBarPolicy">
+    <enum>Qt::ScrollBarAsNeeded</enum>
+   </property>
+   <property name="horizontalScrollBarPolicy">
+    <enum>Qt::ScrollBarAsNeeded</enum>
+   </property>
+   <property name="markdown">
+    <string/>
+   </property>
+   <property name="html">
+    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
+&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
+p, li { white-space: pre-wrap; }
+hr { height: 1px; border-width: 0; }
+li.unchecked::marker { content: &quot;\2610&quot;; }
+li.checked::marker { content: &quot;\2612&quot;; }
+&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
+&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+   </property>
+   <property name="openExternalLinks">
+    <bool>true</bool>
+   </property>
+  </widget>
  </widget>
  <resources>
   <include location="../resources/resources.qrc"/>

--- a/src/usdb_syncer/gui/forms/AboutDialog.ui
+++ b/src/usdb_syncer/gui/forms/AboutDialog.ui
@@ -59,14 +59,73 @@ padding: 10px;</string>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
-   <property name="verticalScrollBarPolicy">
-    <enum>Qt::ScrollBarAsNeeded</enum>
-   </property>
    <property name="horizontalScrollBarPolicy">
-    <enum>Qt::ScrollBarAsNeeded</enum>
+    <enum>Qt::ScrollBarAlwaysOff</enum>
+   </property>
+   <property name="verticalScrollBarPolicy">
+    <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="markdown">
-    <string/>
+    <string>
+
+
+**Thank you** for using USDB Syncer!
+
+Keep your karaoke song collection up to
+date at the click of a button.
+
+***
+
+**More info**
+
+[USDB Syncer Wiki](https://github.com/bohning/usdb_syncer/wiki)
+
+
+***
+
+**Support
+You can support the developers by buying them some 
+[vegan pizza!](https://www.buymeacoffee.com/usdbsyncer)
+
+
+***
+
+**Main Programmers
+bohning
+Ultorex
+
+**Contributing Programmers**
+mjhalwa
+g3n35i5
+BWagener
+
+***
+
+**Application Icon
+rawpixel.com on Freepik
+Website: 
+[freepik.com](https://www.freepik.com/free-vector/pink-neon-cloud-icon-digital-networking-system_16406257.htm)
+**Fugue Icons**
+© 2021 Yusuke Kamiyamane
+Website: 
+[p.yusukekamiyamane.com/](https://p.yusukekamiyamane.com/)
+License: 
+[CC Attribution 3.0 Unported](http://creativecommons.org/licenses/by/3.0)
+
+***
+
+
+**Tester**
+Rakuri
+Hoanzl
+GrüneNeun
+
+
+
+
+**Copyright © 2023**
+
+</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
@@ -75,8 +134,9 @@ p, li { white-space: pre-wrap; }
 hr { height: 1px; border-width: 0; }
 li.unchecked::marker { content: &quot;\2610&quot;; }
 li.checked::marker { content: &quot;\2612&quot;; }
-&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
-&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
+&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;br /&gt;&lt;br /&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Thank you&lt;/span&gt; for using USDB Syncer!&lt;br /&gt;&lt;br /&gt;Keep your karaoke song collection up to date at the click of a button.&lt;br /&gt;&lt;br /&gt;***&lt;br /&gt;&lt;br /&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;More info&lt;/span&gt;&lt;br /&gt;&lt;a href=&quot;https://github.com/bohning/usdb_syncer/wiki&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;USDB Syncer Wiki&lt;/span&gt;&lt;/a&gt;&lt;br /&gt;&lt;br /&gt;&lt;br /&gt;***&lt;br /&gt;&lt;br /&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Support&lt;/span&gt;&lt;br /&gt;You can support the developers by buying them some &lt;a href=&quot;https://www.buymeacoffee.com/usdbsyncer&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;vegan pizza!&lt;/span&gt;&lt;/a&gt;&lt;br /&gt;&lt;br /&gt;&lt;br /&gt;***&lt;br /&gt;&lt;br /&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Main Programmers&lt;/span&gt;&lt;br /&gt;bohning&lt;br /&gt;Ultorex&lt;br /&gt;&lt;br /&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Contributing Programmers&lt;/span&gt;&lt;br /&gt;mjhalwa&lt;br /&gt;g3n35i5&lt;br /&gt;BWagener&lt;br /&gt;&lt;br /&gt;***&lt;br /&gt;&lt;br /&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Application Icon&lt;/span&gt;&lt;br /&gt;rawpixel.com on Freepik&lt;br /&gt;Website: &lt;a href=&quot;https://www.freepik.com/free-vector/pink-neon-cloud-icon-digital-networking-system_16406257.htm&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;freepik.com&lt;/span&gt;&lt;/a&gt;&lt;br /&gt;&lt;br /&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Fugue Icons&lt;/span&gt;&lt;br /&gt;© 2021 Yusuke Kamiyamane&lt;br /&gt;Website: &lt;a href=&quot;https://p.yusukekamiyamane.com/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;p.yusukekamiyamane.com/&lt;/span&gt;&lt;/a&gt;&lt;br /&gt;License: &lt;a href=&quot;http://creativecommons.org/licenses/by/3.0&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;CC Attribution 3.0 Unported&lt;/span&gt;&lt;/a&gt;&lt;br /&gt;&lt;br /&gt;***&lt;br /&gt;&lt;br /&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Tester&lt;/span&gt;&lt;br /&gt;Rakuri&lt;br /&gt;Hoanzl&lt;br /&gt;GrüneNeun&lt;br /&gt;&lt;br /&gt;&lt;br /&gt;&lt;br /&gt;&lt;br /&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Copyright © 2023&lt;/span&gt;&lt;/p&gt;
+&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'.AppleSystemUIFont'; font-size:13pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>

--- a/src/usdb_syncer/gui/forms/AboutDialog.ui
+++ b/src/usdb_syncer/gui/forms/AboutDialog.ui
@@ -37,10 +37,10 @@
   <widget class="QTextBrowser" name="credits">
    <property name="geometry">
     <rect>
-     <x>30</x>
-     <y>130</y>
-     <width>281</width>
-     <height>91</height>
+     <x>20</x>
+     <y>20</y>
+     <width>211</width>
+     <height>201</height>
     </rect>
    </property>
    <property name="sizePolicy">
@@ -151,6 +151,48 @@ li.checked::marker { content: &quot;\2612&quot;; }
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
+   </property>
+  </widget>
+  <widget class="QLabel" name="label_version">
+   <property name="geometry">
+    <rect>
+     <x>0</x>
+     <y>0</y>
+     <width>428</width>
+     <height>140</height>
+    </rect>
+   </property>
+   <property name="styleSheet">
+    <string notr="true">background-image: url(); 
+color: rgb(0, 174, 238);
+font: 24pt &quot;Kozuka Gothic Pro&quot;;</string>
+   </property>
+   <property name="text">
+    <string>VERSION</string>
+   </property>
+   <property name="alignment">
+    <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
+   </property>
+  </widget>
+  <widget class="QLabel" name="label_commit">
+   <property name="geometry">
+    <rect>
+     <x>0</x>
+     <y>0</y>
+     <width>428</width>
+     <height>155</height>
+    </rect>
+   </property>
+   <property name="styleSheet">
+    <string notr="true">background-image: url(); 
+color: rgb(0, 174, 238);
+font: 12pt &quot;Kozuka Gothic Pro&quot;;</string>
+   </property>
+   <property name="text">
+    <string>COMMIT HASH</string>
+   </property>
+   <property name="alignment">
+    <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
    </property>
   </widget>
  </widget>

--- a/src/usdb_syncer/gui/forms/MainWindow.ui
+++ b/src/usdb_syncer/gui/forms/MainWindow.ui
@@ -671,9 +671,10 @@
    </widget>
    <widget class="QMenu" name="menu_about">
     <property name="title">
-     <string>About</string>
+     <string>Help</string>
     </property>
     <addaction name="action_show_log"/>
+    <addaction name="action_about"/>
    </widget>
    <widget class="QMenu" name="menu_songs">
     <property name="title">
@@ -701,7 +702,7 @@
      <normaloff>:/icons/tag-hash.png</normaloff>:/icons/tag-hash.png</iconset>
    </property>
    <property name="text">
-    <string>Create Meta Tags</string>
+    <string>Create Meta Tags...</string>
    </property>
   </action>
   <action name="action_settings">
@@ -710,7 +711,7 @@
      <normaloff>:/icons/cog.png</normaloff>:/icons/cog.png</iconset>
    </property>
    <property name="text">
-    <string>Settings</string>
+    <string>Settings...</string>
    </property>
   </action>
   <action name="action_generate_song_pdf">
@@ -737,7 +738,7 @@
      <normaloff>:/icons/database.png</normaloff>:/icons/database.png</iconset>
    </property>
    <property name="text">
-    <string>Find Local Songs</string>
+    <string>Find Local Songs...</string>
    </property>
   </action>
   <action name="action_show_log">
@@ -746,7 +747,7 @@
      <normaloff>:/icons/log.png</normaloff>:/icons/log.png</iconset>
    </property>
    <property name="text">
-    <string>Show Log</string>
+    <string>Show Log...</string>
    </property>
   </action>
   <action name="action_songs_download">
@@ -792,7 +793,22 @@
      <normaloff>:/icons/faviconUSDB.ico</normaloff>:/icons/faviconUSDB.ico</iconset>
    </property>
    <property name="text">
-    <string>USDB Login</string>
+    <string>USDB Login...</string>
+   </property>
+  </action>
+  <action name="actionHelp">
+   <property name="text">
+    <string>Help...</string>
+   </property>
+  </action>
+  <action name="action_about">
+   <property name="text">
+    <string>About...</string>
+   </property>
+  </action>
+  <action name="actionHelp_2">
+   <property name="text">
+    <string>Help...</string>
    </property>
   </action>
  </widget>

--- a/src/usdb_syncer/gui/mw.py
+++ b/src/usdb_syncer/gui/mw.py
@@ -17,6 +17,7 @@ from PySide6.QtWidgets import (
 )
 
 from usdb_syncer import settings
+from usdb_syncer.constants import VERSION
 from usdb_syncer.gui.debug_console import DebugConsole
 from usdb_syncer.gui.ffmpeg_dialog import check_ffmpeg
 from usdb_syncer.gui.forms.MainWindow import Ui_MainWindow
@@ -331,14 +332,13 @@ def generate_splashscreen() -> QSplashScreen:
     font.setFamily("Kozuka Gothic Pro")
     font.setPointSize(24)
     painter.setFont(font)
-    version = "0.1.0"
     painter.drawText(
         0,
         0,
         428,
         140,
         Qt.AlignmentFlag.AlignRight | Qt.AlignmentFlag.AlignBottom,
-        version,
+        VERSION,
     )
     painter.end()
     return QSplashScreen(canvas)

--- a/src/usdb_syncer/gui/mw.py
+++ b/src/usdb_syncer/gui/mw.py
@@ -17,7 +17,7 @@ from PySide6.QtWidgets import (
 )
 
 from usdb_syncer import settings
-from usdb_syncer.constants import VERSION
+from usdb_syncer.constants import COMMIT_HASH, VERSION
 from usdb_syncer.gui.about_dialog import AboutDialog
 from usdb_syncer.gui.debug_console import DebugConsole
 from usdb_syncer.gui.ffmpeg_dialog import check_ffmpeg
@@ -341,6 +341,16 @@ def generate_splashscreen() -> QSplashScreen:
         140,
         Qt.AlignmentFlag.AlignRight | Qt.AlignmentFlag.AlignBottom,
         VERSION,
+    )
+    font.setPointSize(12)
+    painter.setFont(font)
+    painter.drawText(
+        0,
+        0,
+        428,
+        155,
+        Qt.AlignmentFlag.AlignRight | Qt.AlignmentFlag.AlignBottom,
+        COMMIT_HASH,
     )
     painter.end()
     return QSplashScreen(canvas)

--- a/src/usdb_syncer/gui/mw.py
+++ b/src/usdb_syncer/gui/mw.py
@@ -18,6 +18,7 @@ from PySide6.QtWidgets import (
 
 from usdb_syncer import settings
 from usdb_syncer.constants import VERSION
+from usdb_syncer.gui.about_dialog import AboutDialog
 from usdb_syncer.gui.debug_console import DebugConsole
 from usdb_syncer.gui.ffmpeg_dialog import check_ffmpeg
 from usdb_syncer.gui.forms.MainWindow import Ui_MainWindow
@@ -101,6 +102,7 @@ class MainWindow(Ui_MainWindow, QMainWindow):
             (self.action_usdb_login, lambda: UsdbLoginDialog(self).show()),
             (self.action_meta_tags, lambda: MetaTagsDialog(self).show()),
             (self.action_settings, lambda: SettingsDialog(self).show()),
+            (self.action_about, lambda: AboutDialog(self).show()),
             (self.action_generate_song_pdf, self._generate_song_pdf),
             (self.action_show_log, lambda: open_file_explorer(AppPaths.log)),
         ):

--- a/src/usdb_syncer/gui/song_table/song_table.py
+++ b/src/usdb_syncer/gui/song_table/song_table.py
@@ -232,6 +232,8 @@ class SongTable:
 
     def _on_download_started(self, song_id: SongId) -> None:
         if not (row := self._model.rows.get(song_id)):
+            logger = get_logger(__file__, song_id)
+            logger.error("Unknown id. Ignoring download start signal.")
             return
         data = self._model.songs[row]
         data.status = DownloadStatus.DOWNLOADING
@@ -242,7 +244,8 @@ class SongTable:
         logger = get_logger(__file__, result.song_id)
         if result.error is not None:
             if (row := self._model.row_for_id(result.song_id)) is None:
-                raise Exception(f"Unexpected id: {result.song_id}!")
+                logger.error("Unknown id. Ignoring download finish signal.")
+                return
             if result.error is DownloadErrorReason.NOT_FOUND:
                 self._model.remove_row(row)
                 logger.info("Removed song from local database.")

--- a/src/usdb_syncer/settings.py
+++ b/src/usdb_syncer/settings.py
@@ -18,7 +18,10 @@ import keyring
 from PySide6.QtCore import QByteArray, QSettings
 
 from usdb_syncer.constants import Usdb
+from usdb_syncer.logger import get_logger
 from usdb_syncer.typing_helpers import assert_never
+
+_logger = get_logger(__file__)
 
 SYSTEM_USDB = "USDB Syncer/USDB"
 
@@ -217,7 +220,11 @@ class Browser(Enum):
                 function = browser_cookie3.vivaldi
             case _ as unreachable:
                 assert_never(unreachable)
-        return function(domain_name=Usdb.DOMAIN)
+        try:
+            return function(domain_name=Usdb.DOMAIN)
+        except browser_cookie3.BrowserCookieError:
+            _logger.debug(f"Failed to retrieve {str(self).capitalize()} cookies.")
+        return None
 
 
 class VideoContainer(Enum):

--- a/src/usdb_syncer/song_txt/headers.py
+++ b/src/usdb_syncer/song_txt/headers.py
@@ -18,7 +18,7 @@ from usdb_syncer.song_txt.tracks import replace_false_apostrophes_and_quotation_
 class BeatsPerMinute:
     """New type for beats per minute float."""
 
-    value: float
+    value: float = NotImplemented
 
     def __str__(self) -> str:
         return f"{round(self.value, 2):g}"

--- a/src/usdb_syncer/sync_meta.py
+++ b/src/usdb_syncer/sync_meta.py
@@ -18,6 +18,13 @@ SYNC_META_VERSION = 1
 _logger = get_logger(__file__)
 
 
+class SyncMetaTooNewError(Exception):
+    """Raised when trying to decode meta info from an incompatible future release."""
+
+    def __str__(self) -> str:
+        return "cannot read sync meta written by a future release"
+
+
 @attrs.define
 class FileMeta:
     """Meta data about a local file."""
@@ -70,7 +77,7 @@ class SyncMeta:
     @classmethod
     def from_dict(cls, dct: Any) -> SyncMeta:
         if int(dct["version"]) > SYNC_META_VERSION:
-            raise Exception("cannot read data written by a later version")
+            raise SyncMetaTooNewError
         return cls(
             SongId(dct["song_id"]),
             meta_tags=MetaTags.parse(dct["meta_tags"], _logger),

--- a/tox.ini
+++ b/tox.ini
@@ -1,11 +1,11 @@
 [tox]
-envlist = py310-{lint, test}
+envlist = py311-{lint, test}
 isolated_build = True
 
 [gh-actions]
-python = 3.10: py310-{lint, test}
+python = 3.11: py311-{lint, test}
 
-[testenv:py310-lint]
+[testenv:py311-lint]
 description = Run static checkers.
 extras = lint
 commands =
@@ -25,7 +25,7 @@ commands =
     # pylint --load-plugins pylint.extensions.docparams src setup.py {posargs}
 
 
-[testenv:py310-test]
+[testenv:py311-test]
 description = Run doc tests and unit tests.
 extras = test
 commands =

